datalab/datalab.sh

```diff
@@ -149,8 +150,9 @@ EOF
 }
 
 function run_datalab() {
-  if docker run -d --restart always --net=host \
-    -v "${DATALAB_DIR}:/content/datalab" ${VOLUME_FLAGS} datalab-pyspark; then
+  if docker run -d --restart always -p "${DATALAB_PORT}:8080" \
```

Comment:
Delete `:8080`

Reply:
It is not a mistake, it is a port mapping.